testdata/ansible/memcached-operator/bundle/manifests/cache.example.com_memcacheds.yaml

```diff
@@ -1,50 +0,0 @@
-apiVersion: apiextensions.k8s.io/v1
```

Comment:
Why are many of the files under the `manifests/` directory getting removed for the ansible testdata?